modules/contact-form/grunion-contact-form.php

```diff
@@ -2026,9 +2026,22 @@ static function parse( $attributes, $content ) {
 			 *
 			 * @param string $class Additional CSS classes for button attribute.
 			 */
-			$submit_button_class = apply_filters( 'jetpack_contact_form_submit_button_class', 'pushbutton-wide' );
+			$submit_button_class = apply_filters( 'jetpack_contact_form_submit_button_class', 'pushbutton-wide jetpack-contact-form-button' );
 
-			$r .= "\t\t<input type='submit' value='" . esc_attr( $form->get_attribute( 'submit_button_text' ) ) . "' class='" . $submit_button_class . "'/>\n";
+			$submit_button_styles = '';
+			if ( isset( $attributes['customBackgroundButtonColor'] ) && $attributes['customBackgroundButtonColor'] ) {
+				$submit_button_styles .= 'background-color: ' . $attributes['customBackgroundButtonColor'] . '; ';
+			}
+			if ( isset( $attributes['customTextButtonColor'] ) && $attributes['customTextButtonColor'] ) {
+				$submit_button_styles .= 'color: ' . $attributes['customTextButtonColor'] . ';';
+			}
+			if ( isset( $attributes['submitButtonText'] ) && $attributes['submitButtonText'] ) {
+				$submit_button_text = $attributes['submitButtonText'];
+			} else {
+				$submit_button_text = $form->get_attribute( 'submit_button_text' );
+			}
+
+			$r .= "\t\t<input type='submit' value='" . esc_attr( $submit_button_text ) . "' class='" . $submit_button_class . "' style='" . esc_attr( $submit_button_styles ) . "'/>\n";
```

Comment:
I see that `$submit_button_class` was not escaped before, but should it be?

Reply:
While the classes aren't user submitted, I don't think it can hurt. Done.